packages/jupyterlab-voila/src/index.ts

```diff
@@ -66,7 +68,7 @@ class VoilaRenderButton
     const button = new ToolbarButton({
       className: "voilaRender",
       tooltip: "Render with Voila",
-      iconClassName: VOILA_ICON_CLASS,
+      iconClass: VOILA_ICON_CLASS,
```

Comment:
Good thing the icons still support this :+1: 

We could update to `LabIcon` at a later stage (in a separate PR).

Reply:
Yeah I have been suprised it was still working fine with that. Upgrading to `LabIcon` also popped up in my mind for another PR.

Reply:
It was fixed in https://github.com/jupyterlab/jupyterlab/pull/7947